orchagent/vnetorch.cpp

```diff
@@ -3157,3 +3167,240 @@ bool MonitorOrch::delOperation(const Request& request)
 
     return true;
 }
+
+TunnelTermHelper::TunnelTermHelper(DBConnector *cfgDb)
```

Comment:
We should have a separate PR in the future to re-use these functions in DashEniFwdOrch. Can the class TunnelTermHelper be moved into a different .cpp/.h file. It would be clear that way